### PR TITLE
update glowsticks' battery and time symbols to non mdi- symbols which were removed

### PR DESCRIPTION
**Pull request by bend-n**
### Prerequisites

- [x] I have read and understood the [contributing guide][CONTRIBUTING.md].
- [x] The commit message follows the [conventional commits][cc] guidelines.

### Description

image

also lets me add more increments for the charging characters.

[CONTRIBUTING.md]: https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/CONTRIBUTING.md
[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

**Comment by bend-n** [comment]
Also, why doesnt the rprompt render? 
image

**Comment by JanDeDobbeleer** [comment]
@bend-n some characters don't really work well with the built-in renderer. But that goes way beyond the scope of this project for me to fix.

File: themes/glowsticks.omp.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ blocks:
           always_enabled: true
           style: round
         style: powerline
-        template: " 羽{{ .FormattedMs }} "
+        template: " 󰔟 {{ .FormattedMs }} "
```
Comment (JanDeDobbeleer) [inline themes/glowsticks.omp.yaml:55]: we need the UTF notation of the icons instead of the actual icon: `\uXXXX`

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: themes/glowsticks.omp.yaml (hunk 1 above)
Comment (bend-n) [reply themes/glowsticks.omp.yaml:55]: why? for all of them?

Comment (JanDeDobbeleer) [reply themes/glowsticks.omp.yaml:55]: This isn't readable. When you export the config again that "should" be done for you btw.

Comment (bend-n) [reply themes/glowsticks.omp.yaml:55]: I used `omp config export` and this happened:
image

**Pull request closed without merging**